### PR TITLE
feat: feed

### PR DESCRIPTION
**Pull request by wjbright**

### CODE REVIEW

**Review by gold-olar — APPROVED** [review]
Great work... I think we need to find a way to remove the scroll bars. 
For the next iteration of this, we need to
1. Make this component of its own
2. Make this component fetch data endlessly like a werey 
3. Make it scroll faster if possible. It seems a little sluggish atm.

**Comment by wjbright** [comment]
I agree... I will implement an infinite scroll and fetch system with an external api till ours is ready. Still looking into how to improve the scroll speed

File: src/utils/constants.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const RANDOM_URL = "https://randomuser.me/portraits";
+export const CLOUDINARY_URL = "https://res.cloudinary.com";
```
Comment (wjbright) [inline src/utils/constants.ts:2]: @gold-olar

Testing this feature

Comment (wjbright) [reply src/utils/constants.ts:2]: Reply and let me know

**Comment by wjbright** [comment]
@gold-olar Kindly review this PR and merge

**Pull request merged**